Source/ACE.DatLoader/Entity/AnimationHooks/CallPESHook.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ACE.DatLoader.Entity.AnimationHooks
+{
+    public class CallPESHook : IHook
+    {
+        public uint PES { get; set; }
+        public float Pause { get; set; }
+
```

Comment:
both of these should be "private set".  That's going to apply to almost all of these hook sub-types so I won't spam you with comments about it.  Reason being is that no other class should be changing these values.